recipes/r-pander/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = '0.6.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-pander
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: pander_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/pander_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/pander/pander_{{ version }}.tar.gz
+  sha256: f79e4c1f807d9c6b257c79c61978939b938c0fdd55875524274f3a1d43b3b3ab
+
+build:
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-rcpp
+    - r-digest
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+  run:
+    - r-base
+    - r-rcpp
+    - r-digest
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('pander')"  # [not win]
+    - "\"%R%\" -e \"library('pander')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=pander
+  license: AGPL-3
+  summary: Contains some functions catching all messages, 'stdout' and other useful information
+    while evaluating R code and other helpers to return user specified text elements
+    (like
```

Comment:
This looks wrong.

Reply:
Fixed; ```conda skeleton cran``` seems to have stumbled over the ":" in the CRAN summary.